README.md

```diff
@@ -178,7 +178,17 @@ az group create -n $resourceGroupName --location $location
 # Create an IoT Hub, create a consumer group, add a device, and get the device connection string
 az iot hub create -n $iotHubName -g $resourceGroupName --location $location --sku S1
 az iot hub consumer-group create -n $consumerGroupName --hub-name $iotHubName -g $resourceGroupName
-az iot hub show-connection-string -n $iotHubName -g $resourceGroupName
+
+# There is no way to get a built event hub-compatible connection string using the CLI,
+# however the pieces are available with the 'show-connection-string' and 'policy' commands.
+# Alternatively, one can look in the Azure Portal, under 'Build-in endpoints' to copy the value.
+# The format needs to be:
+# Endpoint=<eventHubEndpoints.events.endpoint>;SharedAccessKeyName=iothubowner;SharedAccessKey=<iot hub connection string's SharedAccessKey>;EntityPath=<eventHubEndpoints.events.path>
+az iot hub show --query properties.eventHubEndpoints.events.endpoint --name {YourIoTHubName}
+az iot hub policy show --name service --query primaryKey --hub-name {YourIoTHubName}
+az iot hub show --query properties.eventHubEndpoints.events.path --name {YourIoTHubName}

```

Comment:
This is a lot of work to get the EH conn string.  Pushing user just to use the portal is the easiest bet.

Reply:
I agree, but the docs team hates including screenshots to the portal which changes all the time. My comment says you can get it from the portal, but I include the az cli commands to enable someone who wants to use that.